openshift/openshift-copies.go

```diff
@@ -272,13 +281,14 @@ func getUserIdentificationPartialConfig(configAuthInfo clientcmdAuthInfo) (*rest
 	return mergedConfig, nil
 }
 
-// canIdentifyUser is a modified copy of k8s.io/kubernetes/pkg/client/unversioned/clientcmd.canIdentifyUser
-func canIdentifyUser(config restConfig) bool {
-	return len(config.Username) > 0 ||
-		(len(config.CertFile) > 0 || len(config.CertData) > 0) ||
-		len(config.BearerToken) > 0
+// Unused
+// // canIdentifyUser is a modified copy of k8s.io/kubernetes/pkg/client/unversioned/clientcmd.canIdentifyUser
+// func canIdentifyUser(config restConfig) bool {
+// 	return len(config.Username) > 0 ||
+// 		(len(config.CertFile) > 0 || len(config.CertData) > 0) ||
+// 		len(config.BearerToken) > 0
 
-}
+// }
```

Comment:
I'd prefer removing unused code unless you think there's a very real chance we'll need it again in the not too distant future.